connectors/webhook/src/main/java/io/camunda/connector/inbound/HttpWebhookExecutable.java

```diff
@@ -113,6 +128,38 @@ private boolean webhookSignatureIsValid(
     return true;
   }
 
+  private byte[] extractTwilioSignatureData(final WebhookProcessingPayload payload)
+      throws IOException {
+
+    if (payload.method().equalsIgnoreCase("GET")) {
```

Comment:
If `hmac` is set we can always extract the data from the url. No need to have a custom twilio logic here.